app/src/test/java/org/simple/clinic/patient/PatientRepositoryTest.kt

```diff
@@ -36,20 +39,35 @@ class PatientRepositoryTest {
   private lateinit var ageFuzzer: AgeFuzzer
   private lateinit var searchPatientByName: SearchPatientByName
 
-  private val database = mock<AppDatabase>()
   private val clock = TestClock()
-  private val patientSearchResultDao = mock<PatientSearchResult.RoomDao>()
-  private val patientDao = mock<Patient.RoomDao>()
-  private val patientAddressDao = mock<PatientAddress.RoomDao>()
-  private val patientPhoneNumberDao = mock<PatientPhoneNumber.RoomDao>()
-  private val fuzzyPatientSearchDao = mock<PatientFuzzySearch.PatientFuzzySearchDao>()
-  private val dobValidator = mock<DateOfBirthFormatValidator>()
-  private val userSession = mock<UserSession>()
-  private val facilityRepository = mock<FacilityRepository>()
-  private val numberValidator = mock<PhoneNumberValidator>()
+
+  lateinit var database: AppDatabase
+  lateinit var patientSearchResultDao: PatientSearchResult.RoomDao
+  lateinit var patientDao: Patient.RoomDao
+  lateinit var patientAddressDao: PatientAddress.RoomDao
+  lateinit var patientPhoneNumberDao: PatientPhoneNumber.RoomDao
+  lateinit var fuzzyPatientSearchDao: PatientFuzzySearch.PatientFuzzySearchDao
+  lateinit var dobValidator: DateOfBirthFormatValidator
+  lateinit var userSession: UserSession
+  lateinit var facilityRepository: FacilityRepository
+  lateinit var numberValidator: PhoneNumberValidator
```

Comment:
We can make all of these properties `private`.

Reply:
Done.